PocketExample/Pocket/Models/Network.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Network.swift
+//  Pocket
+//
+//  Created by Pabel Nunez Landestoy on 4/25/18.
+//  Copyright © 2018 Pocket Network. All rights reserved.
+//
+
+import Foundation
+
+class Network {
+    var tokenID: String
+    init(tokenID: String) {
```

Comment:
If you turn this into a struct. you don't need to write the initializer explicitly as Swift will create the struct using member-wise initialization (https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Initialization.html) - scroll down to the `Memberwise Initializers for Structure Types` section.